src/selectors/per-thread/markers.js

```diff
@@ -94,21 +97,51 @@ export function getMarkerSelectorsPerThread(threadSelectors: *) {
       )
   );
 
-  const getCommittedRangeFilteredMarkers: Selector<Marker[]> = createSelector(
+  const getMarkerGetter: Selector<(MarkerIndex) => Marker> = createSelector(
```

Comment:
Blocking: I don't think this new mechanic is not documented enough. At the very least we should document this selector, and make sure the big comment above gets updated somewhat. This new mechanic is a bit surprising, we should explain why it exists the way it does.